README.md

```diff
@@ -117,7 +119,8 @@ For Mobile QQ only, the internal browser does not support the identify of qrcode
 
 For Alipay only, user will be redirect to the payment link directly. No need for second-time scaning.
 
-## Contribute
+## Contributions
+
```

Comment:
Personally, contribution means contribute works that have been done. But this section is all about encouraging people to contribute to this project. So I would think "Contributions" should not be used here.